torch/utils/tensorboard/__init__.py

```diff
@@ -0,0 +1,6 @@
+try:
```

Comment:
Please only wrap tensorboard with the import check and remove some of the extra prints

```
try:
    from tensorboard.summary.writer.record_writer import RecordWriter  # noqa F401
except ImportError:
    raise ImportError('TensorBoard logging requires TensorBoard with Python summary writer installed. '
                                  'This should be available in 1.14 or above.')

from .writer import FileWriter, SummaryWriter  # noqa F401
```

Reply:
btw, I think the version should > 1.14 after it's landed.